docs/developers/api.md

```diff
@@ -0,0 +1,223 @@
+
+
+- `organisations` - `object`: 
+    - `id` - `string`
+    - `name` - `string`
+    - `role` - `string`
+- `organisation` - `string`: maps to `organisations.name`
+
+
+- `publications` - `object`: 
+    - `id` - `string`
+    - `title` - `string`
+- `publication` - `string`: maps to `publications.title`
+
+
+- `documents` - `object`:
+    - `id` - `string`
+    - `name` - `string`
+    - `source_id` - `string`
+    - `source_url` - `string`
+    - `type` - `string`
+    
+    - `fda_approval` - `object`:
+        - `id` - `string`
+        - `action_date` - `date`
+        - `notes` - `string`
+        - `supplement_number` - [`long`](https://www.elastic.co/guide/en/elasticsearch/reference/current/number.html)
+        - `type` - `string`
+        - `fda_application` - `object`:
+            - `id` - `string`
+            - `type` - `string`
+            - `drug_name` - `string`
+            - `active_ingredients` - `string`
+    -  `file` - `object`:
+        - `id` - `string`
+        - `documentcloud_id` - `string`
+        - `source_url` - `string`
+
+#### FDA documents search
+
+Here is a list of the attributes included in the `FDADocuments` index:
+
+- `id` - `string`
+- `name` - `string`
+- `source_id` - `string`
+- `source_url` - `string`
+- `type` - `string`
+
+- `fda_approval` - `object`:
+    - `id` - `string`
+    - `action_date` - `date`
+    - `notes` - `string`
+    - `supplement_number` - [`long`](https://www.elastic.co/guide/en/elasticsearch/reference/current/number.html)
+    - `type` - `string`
+    - `fda_application` - `object`:
+        - `id` - `string`
+        - `type` - `string`
+        - `drug_name` - `string`
+        - `active_ingredients` - `string`
+
+-  `file` - `object`:
+    - `id` - `string`
+    - `documentcloud_id` - `string`
+    - `source_url` - `string`
+    - `pages` - `string` - raw text gathered with optical character recognition from files
+
+- `trial` - `object`:
+    - `id` - `string`
+    - `public_title` - `string`
+
+#### Examples
+
+* Simple attribute query 
+
+    To get all the `trials` directed at women:
+    ```
+    /v1/search?q=gender:female
+    ```
+* Navigate to the next page
+
+    ```
+    /v1/search?q=gender:female&page=2
+    ```
+
+* Nested attribute query
+
+    To get all the `trials` that research breast cancer:
+    ```
+    /v1/search?q=conditions.name:breast cancer
+    ```
+    Notice that we use `conditions.name` to reference the `name` attribute of `conditions`.
+    The same syntax applies for any level of nesting in attributes of type `object`.
+    
+    For ease of use, we also allowed some shorthand queries for common nested attributes (described in the list with `maps to`).
+    Therefore you can use the following query to get the same results:
+
+    ```
+    /v1/search?q=condition:breast cancer
+    ```
+* Identifiers attribute
+
+    The `identifiers` attribute is just a normal `object` but its attributes are names of identifiers. 
```

Comment:
I wonder if this sounds better:

[...] is a JSON `object` with names of identifiers as attributes (e.g. `nct`).